src/ImportInfoDialog.cpp

```diff
@@ -0,0 +1,39 @@
+#include "ImportInfoDialog.h"
+
+BEGIN_EVENT_TABLE( ImportInfoDialog, wxDialogWrapper )
+   EVT_BUTTON( wxID_CANCEL, ImportInfoDialog::OnCancel )
+END_EVENT_TABLE()
+
+ImportInfoDialog::ImportInfoDialog(const FilePath& filename, wxWindow* parent, wxWindowID id, const TranslatableString &title, const wxPoint &position, const wxSize& size, long style ):
+      wxDialogWrapper( parent, id, title, position, size, style )
+{
+    wxBoxSizer* sizer = new wxBoxSizer(wxVERTICAL);
+
+    m_statusText = new wxStaticText(this, wxID_ANY, wxString::Format(wxT("Importing %s..."), filename.AfterLast(wxFileName::GetPathSeparator())));
```

Comment:
Use either `XO` or `_` macro for translatable strings, otherwise `gettext` will not see your string definition

Reply:
fixed